corehq/apps/linked_domain/view_helpers.py

```diff
@@ -161,6 +181,18 @@ def build_keyword_view_model(keyword, last_update=None):
     )
 
 
+def build_ucr_expression_view_model(ucr_expression, last_update=None):
+    if not ucr_expression:
+        return None
+
+    return build_linked_data_view_model(
+        model_type=MODEL_UCR_EXPRESSION,
+        name=f"{LINKED_MODELS_MAP[MODEL_UCR_EXPRESSION]} ({ucr_expression.name})",
+        detail=UCRExpressionLinkDetail(ucr_expression_id=str(ucr_expression.id)).to_json(),
+        last_update=last_update,
+    )
```

Comment:
There are some tests in `linked_domain/tests/test_view_helpers.py` that test build view model methods. I'll admit it feels a bit tedious, but is also a quick and easy way to ensure view models are built properly.

Reply:
I added more tests there in the same style as the others.